db/migrate/20150507181633_add_default_to_hire_count.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultToHireCount < ActiveRecord::Migration
+  def change
+    remove_column :companies, :hire_count
+    add_column :companies, :hire_count, :integer, default: 0
+  end
+end
```

Comment:
I think our changes to the form do this too.  Nothing wrong with a little duplicaiton / checks and balances.